From 7939a827ab3b1ecb8f7bb75537002bdf061bbd64 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 6 May 2021 10:36:59 +0000
Subject: [PATCH] libstdc++: fix ostream operator<< in LoadStreamTask

std::unique_ptr has no operator<< in libstdc++. Even it would
exist, it would just return the pointer value and not the content.
Referenced objects don't have an operator<< as well, so just
remove them from output stream.
---
 components/feed/core/v2/tasks/load_stream_task.cc | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/components/feed/core/v2/tasks/load_stream_task.cc b/components/feed/core/v2/tasks/load_stream_task.cc
index 4c6dd98..8f328c0 100644
--- a/components/feed/core/v2/tasks/load_stream_task.cc
+++ b/components/feed/core/v2/tasks/load_stream_task.cc
@@ -299,14 +299,11 @@ std::ostream& operator<<(std::ostream& os,
      << " stored_content_age=" << result.stored_content_age
      << " last_added_time=" << result.last_added_time
      << " load_type=" << static_cast<int>(result.load_type)
-     << " update_request=" << result.update_request
      << " request_schedule?=" << result.request_schedule.has_value();
   if (result.network_response_info)
     os << " network_response_info=" << *result.network_response_info;
   return os << " loaded_new_content_from_network="
-            << result.loaded_new_content_from_network
-            << " latencies=" << result.latencies
-            << " upload_actions_result=" << result.upload_actions_result << "}";
+            << result.loaded_new_content_from_network << "}";
 }
 
 }  // namespace feed
-- 
2.26.3

